src/modules.d.ts

```diff
@@ -0,0 +1,380 @@
+declare module "json-schema-deref-sync" {
+    import {
+        JSONSchema6,
+        JSONSchema6Definition,
+        JSONSchema6Definition,
+        JSONSchema6,
+        JSONSchema6,
+        JSONSchema6,
+        JSONSchema6,
```

Comment:
Should not be the same value multiple times

Reply:
Done